ensime_shared/editor.py

```diff
@@ -278,8 +278,8 @@ def to_quickfix_item(self, file_name, line_number, message, tpe):
                 "text": message,
                 "type": tpe}
 
-    def write_quickfix_list(self, qflist):
-        self._vim.command('call setqflist({!s})'.format(qflist))
+    def write_quickfix_list(self, qflist, title):
+        self._vim.command("call setqflist({!s}, 'r', 'Ensime - {}')".format(qflist, title))
```

Comment:
I believe this signature of `setqflist` is an incompatibility between Vim and Neovim. In Vim 8 the third argument is documented as being a dictionary and has this example:

```vim
:call setqflist([], 'r', {'title': 'My search'})
```

It looks like Vim 7 [didn't support it at all][1] and I think giving up Vim 7 support might be a bit premature now, Neovim difference aside.

[1]: http://vimdoc.sourceforge.net/htmldoc/eval.html#setqflist()